pyproject.toml

```diff
@@ -23,7 +23,8 @@ packages = [
 python = "^3.8"
 click = "^8.0.3"
 prettytable = "^3.1.1"
-pysigma = "^0.11.3"
+# pysigma = "^0.11.3""
+pysigma = { git = "https://github.com/SigmaHQ/pySigma", branch = "main" }
```

Comment:
Switch back to the release and use pySigma 0.11.7 😉

Reply:
Yea yea it's a draft PR 😂